hs-bindgen/app/HsBindgen/App/Common.hs

```diff
@@ -414,23 +411,10 @@ withCliTracer ::
   -> (Tracer IO TraceMsg -> IO a)
   -> IO (Maybe a)
 withCliTracer GlobalOpts{..} action' = do
-    ansiColor <- getAnsiColor
     let customLogLevel = case macroWarnings of
           MacroLogInfo    -> DefaultLogLevel
```

Comment:
Cleaner to assemble the list directly. (turn option(s) to list) and then always use `customLogLevelFrom`.